misc/db_schema.yaml

```diff
@@ -1062,6 +1062,31 @@ pollers:
   Indexes:
     PRIMARY: { Name: PRIMARY, Columns: [poller_name], Unique: true, Type: BTREE }
     id: { Name: id, Columns: [id], Unique: false, Type: BTREE }
+poller_cluster:
```

Comment:
I think your branch is a little out of date.  Remove this and users changes below.